web/ui/dashboard/src/app/components/modal/modal.component.ts

```diff
@@ -1,20 +1,48 @@
 import { CommonModule } from '@angular/common';
-import { Component, EventEmitter, HostListener, Input, OnInit, Output, ViewChild } from '@angular/core';
+import { Component, Directive, EventEmitter, HostListener, Input, OnInit, Output } from '@angular/core';
 import { ButtonComponent } from '../button/button.component';
 import { ModalDialogComponent } from '../modal-dialog/modal-dialog.component';
 import { OverlayDirective } from '../overlay/overlay.directive';
 
+
+@Component({
+	selector: 'convoy-modal-header, [convoy-modal-header]',
+	standalone: true,
+	template: `
+		<div class="px-20px pt-20px pb-16px border-y border-y-grey-10 bg-white-100 rounded-tr-16px rounded-tl-16px w-full ">
+			<div class="flex justify-between items-center max-w-[834px] m-auto">
+				<ng-content></ng-content>
+			</div>
+		</div>
+	`
+})
+export class ModalHeaderComponent {
+	constructor() {}
+}
+
+
+@Directive({
+	selector: '[convoy-modal-body]',
+	standalone: true,
+	host: { class: 'm-auto empty:hidden', '[class]': "position === 'full' ? 'max-w-[834px]' : 'w-full p-20px'" }
```

Comment:
We can set these values on the modal component, i.e. the width.

Hmm, except for the padding yea?